open-api/rest-catalog-open-api.py

```diff
@@ -441,6 +441,30 @@ class AssertViewUUID(BaseModel):
     uuid: str
 
 
+class AzureCredentials(BaseModel):
+    account_name: Optional[str] = Field(None, alias='account-name')
+    account_key: Optional[str] = Field(None, alias='account-key')
+    token: Optional[str] = None
+
+
+class AwsCredentials(BaseModel):
+    access_key_id: str = Field(..., alias='access-key-id')
+    secret_access_key: str = Field(..., alias='secret-access-key')
+    token: Optional[str] = None
+    expires_at_ms: Optional[int] = Field(None, alias='expires-at-ms')
+
```

Comment:
```suggestion
    region: Optional[str] = None
    endpoint: Optional[str] = None
```

Reply:
`region` / `endpoint` isn't really credential-related so I don't think these should be included here